src/start/RealmInputScreen.js

```diff
@@ -162,7 +289,7 @@ export default function RealmInputScreen(props: Props): Node {
           ref={textInputRef}
         />
       </View>
-      <ZulipTextIntl text="e.g. zulip.example.com" style={styles.hintText} />
+      {renderedSuggestion}
```

Comment:
Interesting!

One thing I see in the screen recordings is that the input and the submit button bounce up and down a bit as this line appears and disappears. Can we perhaps put a spacer in here when we have nothing to say, so that the layout is stable?

Reply:
That would be nice!